### PR TITLE
[Feature] Task 3: data base and file storage engine corrected

### PR DESCRIPTION
**Pull request by valvarezgi**
Updated engines for task 3

### CODE REVIEW

File: models/engine/db_storage.py [hunk 1]
```diff
@@ -74,3 +74,18 @@ def reload(self):
...
+                return v
+        return None
+
+    def count(self, cls=None):
+        """returns the number of objects in storage matching
+            the given class name"""
+        if cls is None:
+            return len(self.all())
+        else:
+            return len(self.all(cls))
```
Comment (RicardoBarretoR) [inline models/engine/db_storage.py:91]: It is recommended to be more specific with the names of the variables for a better identification 

Do not forget to made the respective tests

**Review by RicardoBarretoR — CHANGES_REQUESTED** [review]

File: models/engine/db_storage.py [hunk 2]
```diff
@@ -74,3 +74,18 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """returns the object based on the class name and its Id"""
+        for k, v in self.all(cls).items():
+            if id in k:
+                return v
```
Comment (RicardoBarretoR) [inline models/engine/db_storage.py:82]: It is recommended to be more specific with the names of the variables for a better identification 

Do not forget to made the respective tests

File: models/engine/file_storage.py [hunk 3]
```diff
@@ -68,3 +68,18 @@ def delete(self, obj=None):
     def close(self):
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
+        """returns the object based on the class name and its Id"""
+        for k, v in self.all(cls).items():
+            if id in k:
+                return v
```
Comment (RicardoBarretoR) [inline models/engine/file_storage.py:76]: It is recommended to be more specific with the names of the variables for a better identification 

Do not forget to made the respective tests

**Review by RicardoBarretoR — CHANGES_REQUESTED** [review]

File: models/engine/db_storage.py (hunk 2 above)
Comment (valvarezgi) [reply models/engine/db_storage.py]: Changes in variables names made. Tests will be added later.

File: models/engine/file_storage.py (hunk 3 above)
Comment (valvarezgi) [reply models/engine/file_storage.py]: Changes in variables names made. Tests will be added later.

**Review by RicardoBarretoR — APPROVED** [review]
ok

**Pull request merged**